### ISSUE TITLE
Verify Proper accessibilityRole for Buttons in Rx

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#8219]
To align with accessibility best practices and with the design system, we should ensure that all button components have an accessibilityRole of "button" so that a screen reader will announce it as such. 

Although we did find a way to swap the accessibilityRole to a link as a result of ticket #7808 (that was created based on the buttons and links audit epic #7754), accessibility best practice is for things that look like buttons to have a role of "button" and things that look like links to have a role of "link". In my initial ticket, I did include an option of overriding the role, but also did not make it clear that this should be an absolute last resort and not something that we do unless there is **no other option available**. 

For feature screen buttons specifically, they should be left as buttons in their styling and role to help keep the existing UX pattern that has been established throughout the app.

**Comment by rbontrager** [comment]
Verified that the buttons in RX now say <start refill request button> when using voiceOver/talkback.  Approved by QA.
